### PR TITLE
Add_t5gemma_model

### PR DESCRIPTION
**Pull request by Minestar6**
# Add T5gemma
添加了t5gemma模型的PaddlePaddle 实现，包含模型代码、配置文件、单元测试。
# 对齐验证
## 精度对齐
前 20 个 token 的 logits 对齐结果如下：
* mean diff: 5.341125e-06
* max diff: 7.390976e-05  
token完全一致

### 训练数据
文本数据
* GSM8K SFT 数据
* 训练步数：300 steps
1_t5gemma
1_t5gemma_diff

### CODE REVIEW

**Comment by risemeup1111** [comment]
## Failed CI看板

| 流水线名称 | 问题标签 | 修复建议 | 日志片段 |
| ------------ | ---------- | ---------- | ---------- |
| Clone-Linux / code-clone | 日志获取失败，无法确定根因 | 重新运行 `CI_XPU`，并获取新的 job 日志后再分析 | [Job](https://github.com/PaddlePaddle/PaddleFormers/actions/runs/33251366655/job/99097500141) |

<details>
<summary>日志分析报告</summary>

**失败的测试 case**:
```text
Job: Clone-Linux / code-clone
测试 case: 日志不可用，无法确认具体测试步骤
关键错误: 日志获取失败：HTTP 404，The specified blob does not exist
```

**根本原因分析**:

当前快照只能确认 `CI_XPU` 的 `Clone-Linux / code-clone` job 失败，但对应日志 Blob 返回 404，未取得任何可验证的执行日志。因此无法判断失败是否由 PR #4936 的代码、CI 基础设施或上游依赖引起，也不能从 PR diff 推断具体根因。

现有证据更直接指向 CI 日志产物不可用或已过期，而不是 PR 代码本身的确定性错误。PR diff 中新增的 T5Gemma 实现暂时没有日志证据证明与该 job 失败相关。

**修复建议**:

1. 重新运行 `CI_XPU`，优先检查 `Clone-Linux / code-clone` 的完整日志。
2. 若重跑仍失败，根据新的具体错误定位 T5Gemma 改动或 CI 环境问题。
3. 若重跑成功，可将本次失败视为暂时性的 CI 日志产物或基础设施异常。

</details>

Powered by Nyanpasu with gpt-5.6-luna 默认推理级别, please check the suggestions carefully.
